ui/apps/platform/src/Containers/Login/LoginPage.js

```diff
@@ -126,20 +157,28 @@ class LoginPage extends Component {
     };
 
     renderFields = () => {
-        const { authStatus, authProviders } = this.props;
+        const { authStatus, authorizeRoxctlMode } = this.props;
+        let { authProviders } = this.props;
         if (
-            authStatus === AUTH_STATUS.LOADING ||
-            authStatus === AUTH_STATUS.LOGGED_IN ||
-            authStatus === AUTH_STATUS.ANONYMOUS_ACCESS
+            !authorizeRoxctlMode &&
+            (authStatus === AUTH_STATUS.LOADING ||
+                authStatus === AUTH_STATUS.LOGGED_IN ||
+                authStatus === AUTH_STATUS.ANONYMOUS_ACCESS)
         ) {
             return null;
         }
 
+        let title = 'Log in to your account';
+        if (authorizeRoxctlMode) {
+            authProviders = authProviders.filter((provider) => provider.type !== 'basic');
```

Comment:
This filtering out of basic mode is because roxctl would then use the admin password and not require login in 'basic' mode, right ?

Reply:
Yeah, essentially `type == basic` means admin password. Since this is a completely separate flow from the backend and the UI auth code, and we already have the `--password` flag, people should use that instead. It's currently not possible to define a different `basic` auth provider.